com.unity.formats.alembic/Runtime/Unity.Formats.Alembic.Runtime.api

```diff
@@ -65,10 +65,13 @@ namespace UnityEngine.Formats.Alembic.Importer
         public float MediaEndTime { get; }
         public float MediaStartTime { get; }
         public string PathToAbc { get; }
-        public UnityEngine.Formats.Alembic.Importer.AlembicStreamSettings Settings { get; }
+        public UnityEngine.Formats.Alembic.Importer.AlembicStreamSettings Settings { get; set; }
         public float StartTime { get; set; }
         public float VertexMotionScale { get; set; }
         public AlembicStreamPlayer() {}
+        public bool LoadFromFile(string newPath);
+        public bool ReloadStream(bool createMissingNodes = False);
```

Comment:
Consider using `[Flags]enum ReloadMode { CreateMissingNodes, RemoveUnusedNodes }` instead of a bool here